index.js

```diff
@@ -113,4 +113,4 @@ function gulpPotomo(customOptions, cb) {
 }
 
 // Exporting the plugin main function
-module.exports = gulpPotomo;
\ No newline at end of file
+module.exports = gulpPotomo;
```

Comment:
It's usually good practice to have the extra line at the end of files, so that it doesn't flag up the lack of newline character (shown as the red character above, but also shown as "No newline at end of file".